BackendTaskPilot/BackendTaskPilot.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+    <PropertyGroup>
+        <TargetFramework>net7.0</TargetFramework>
```

Comment:
net8.0 is currently the latest active version so you could update the project to use that instead. However we have to be mindful of any dependencies and any projects that consume this one to ensure compatibility. Also it's possible that the build pipeline will need to be updated to accommodate the .net8.0 update. Stick any questions you have about CI/CD in the #devops channel